README.md

```diff
@@ -12,4 +12,4 @@ The following functions are imported from `conjCalcFunctions.py`:
  - `conjcalc()` : function to take in a dataframe and add columns for all stages of calculating conjugate points.
  - `calc_mlat_rings()` : function to calculate magnetic graticules for a given latitude and datetime.
```

Comment:
These lines and the line above should be updated to be, e.g., `conjugate_map.calc_mlat_rings`

Reply:
Created issue #11